frontend/src/pages/home/Home.tsx

```diff
@@ -316,11 +306,11 @@ export default function Home(props: { updateTheme: (theme: "system" | "dark" | "
 						<i className={ "fi fi-rr-arrow-right" } style={ { marginRight: "var(--spacing)" } }/>
 						<span>For further information, please have a look at the Wiki linked in our GitHub Repository.</span>
 					</a>
-				</InView>
+				</div>
 			</section>
 			
-			<section className={ "page" } id={ "plans" }>
-				<InView as={ "div" } onChange={ inViewOnChange }>
+			<section className={ "page bg-secondary" } id={ "plans" }>
+				<div>
 					<h2 className={ "page-title fade-in" }>Meet our Plans</h2>
 					<h1 className={ "page-subtitle fade-in" }>Take the next step</h1>
```

Comment:
i18n

Reply:
As Home is not in our requirements at all (and is undergoing many changes anyway), I will not add translation (yet).